addons/mail/static/src/core/common/thread_actions.js

```diff
@@ -7,17 +7,44 @@ import { registry } from "@web/core/registry";
 
 export const threadActionsRegistry = registry.category("mail.thread/actions");
 
-threadActionsRegistry.add("close", {
-    condition(component) {
-        return component.props.chatWindow;
-    },
-    icon: "fa fa-fw fa-close",
-    name: _t("Close Chat Window"),
-    open(component) {
-        component.close();
-    },
-    sequence: 100,
-});
+threadActionsRegistry
+    .add("toggle", {
+        condition(component) {
+            return !component.ui.isSmall && component.props.chatWindow;
+        },
+        icon: "fa fa-fw fa-minus",
+        name: _t("Toggle"),
```

Comment:
Not just "Toggle", but have it dynamically show "Fold Chat Window" or "Open Chat Window" depending on fold state.

Reply:
Doesn't it being too long? I prefer something short and clear. If being dynamic. I would say using `Fold` and `Open`. And the icon should also be dynamic or not.

Reply:
Ok for shorter "Fold"/"Open". Thanks!